src/scripts/eastern_kingdoms/dun_morogh/gnomeregan/gnomeregan.cpp

```diff
@@ -649,11 +654,11 @@ enum
 
     SPELL_EXPLOSION             = 27745, //spell de Clank.
 
-    SAY_KERNOBEE_START          = -1780204,
-    SAY_BOMB_START              = -1780205,
```

Comment:
Script text -1780205 has `type`=1 (Yell), while broadcast text id 3927 which u replace it with currently has `chat_type`=0 (Say), but you do not update the broadcast text in your migration. Is this intentional or an error? If it is intentional, why do you change it to Say, when sniffs confirm its Yell?